data/35e/wizards_of_the_coast/core/universal_rules/ur_abilities.lst

```diff
@@ -917,11 +917,12 @@ Resistance to Sonic		CATEGORY:Special Ability	TYPE:ResistanceSelection	VISIBLE:N
 
 ###Block: Resistances Output
 # Ability Name				Output Name						Category of Ability		Type								Visible	Description																			Aspects
-Resistance to Acid Output		OUTPUTNAME:Resistance to Acid			CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinary.Resistance	VISIBLE:YES	DESC:You may ignore %1 points of Acid damage each time you take Acid damage|AcidResistanceBonus					ASPECT:Resistance|Acid	ASPECT:Ability Benefit|%1|AcidResistanceBonus
-Resistance to Cold Output		OUTPUTNAME:Resistance to Cold			CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinary.Resistance	VISIBLE:YES	DESC:You may ignore %1 points of Cold damage each time you take Cold damage|ColdResistanceBonus					ASPECT:Resistance|Cold	ASPECT:Ability Benefit|%1|ColdResistanceBonus
-Resistance to Fire Output		OUTPUTNAME:Resistance to Fire			CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinary.Resistance	VISIBLE:YES	DESC:You may ignore %1 points of Fire damage each time you take Fire damage|FireResistanceBonus					ASPECT:Resistance|Fire	ASPECT:Ability Benefit|%1|FireResistanceBonus
-Resistance to Electricity Output	OUTPUTNAME:Resistance to Electricity	CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinary.Resistance	VISIBLE:YES	DESC:You may ignore %1 points of Electricity damage each time you take Electricity damage|ElectricityResistanceBonus	ASPECT:Resistance|Electricity	ASPECT:Ability Benefit|%1|ElectricityResistanceBonus
-Resistance to Sonic Output		OUTPUTNAME:Resistance to Sonic		CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinary.Resistance	VISIBLE:YES	DESC:You may ignore %1 points of Sonic damage each time you take Sonic damage|SonicResistanceBonus				ASPECT:Resistance|Sonic	ASPECT:Ability Benefit|%1|SonicResistanceBonus
+Resistance to Acid Output		OUTPUTNAME:Resistance to Acid			CATEGORY:Special Ability	TYPE:SpecialQuality.Extraordinary.Resistance	VISIBLE:YES	DESC:You may ignore %1 points of Acid damage each time you take Acid damage|AcidResistanceBonus					ASPECT:SaveBonus|Acid Resistance %1|AcidResistanceBonus	ASPECT:Resistance|Acid	ASPECT:Ability Benefit|%1|AcidResistanceBonus
+# DG - not seeing the original ASPECT features output anywhere, so adding additional
```

Comment:
Just how many times do you need a resistance displayed? I think three times is a bit excessive for the PDF export. 

You might want to add the same functions of the pdf to the htm export for a consistent appearance.